src/mesh/Draco_Mesh_Reader.hh

```diff
@@ -0,0 +1,48 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   mesh/Draco_Mesh_Reader.hh
+ * \author Ryan Wollaeger <wollaeger@lanl.gov>
+ * \date   Friday, Jul 13, 2018, 13:48 pm
+ * \brief  RTT_Draco_Mesh_Reader header file.
+ * \note   Copyright (C) 2018 Los Alamos National Security, LLC.
+ *         All rights reserved. */
+//---------------------------------------------------------------------------//
+
+#ifndef rtt_mesh_Draco_Mesh_Reader_hh
+#define rtt_mesh_Draco_Mesh_Reader_hh
+
+#include <vector>
```

Comment:
@KineticTheory It seems the vector header is being included here.  Should the cstddef header be included here as well (https://en.cppreference.com/w/cpp/types/size_t)?  This is evidently the lightest-weight one to include.

Reply:
Yes. Try that.  Usually including `<vector>` is sufficient.  Another option is to replace `size_t` with `unsigned int`.

Reply:
The header has been included.